referenceframe/testjson/missinglink.json

```diff
@@ -1,24 +1,6 @@
 {
     "name": "missinglink",
     "links": [
-        {
```

Comment:
Why are we changing this?

Reply:
This relates to the other comment I left.  After making this change the test was no longer failing because this does actually specify a valid kinematic chain.  So I am deleting a frame so that there is actually a missing link

Reply:
Never mind; read the above comment